src/HealthCheckServer.ts

```diff
@@ -78,7 +78,8 @@ const failAction = async (_request: Hapi.Request, _handler: Hapi.ResponseToolkit
  */
 
 const createHealthCheckServer = async (port: string, healthCheckHandler: Lifecycle.Method): Promise<Hapi.Server> => {
-  //@ts-ignore - type defs are wrong
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
This seems a little crazy - do we want to disable this rule globally? If not, then I'm fine with it.

Reply:
I would recommend not doing that. I would rather have individuals make the decision to add the annotations as per the example above indicating that the typedefs are missing or invalid etc. The linter annotation would also need to be added.